site/data/docs/connect-button.mdx

```diff
@@ -59,6 +66,16 @@ The `ConnectButton` component exposes several props to customize its appearance
 
 ### Examples
 
+#### Label
+
+By default your Connect Button will say "Connect Wallet".
```

Comment:
in order to keep the doc copy as minimal as possible, I think we could remove this line since it's already documented in the props table above